tools/cloud-build/daily-tests/blueprints/kueue-config-files/tas-queues.yaml

```diff
@@ -44,7 +48,7 @@ spec:
     - name: "tas-flavor"
       resources:
       - name: "nvidia.com/gpu"
-        nominalQuota: 12        # 6 nodes, 2 GPU each
+        nominalQuota: 10000000  # infinite quota
```

Comment:
can you please add a comment how infinite quota is okay for TAS enabled kueue

Reply:
Had discussed this with the Kueue team. The answer given is that unlike non TAS workloads, TAS workloads check the current physical infrastructure layout and the current infrastructure usage of the infrastructure. As a result, it is recommended to set this number really high so that Kueue doesn't stop a workload based on quota but rather, it relies on topology information of available nodes and tries to match it with the workloads requirements. 

If you are a more sophisticated customer, you can do the math and configure the numbers to align with the capacity you have reserved. 

Kueue team is currently working on drafting a document that explains this well. It would share details on how you can do the math. Would it be fine to add a link to it once it is available for customers? 

This file is not used for non-testing purposes in cluster toolkit today.

Reply:
Will update the readme files for A3U when Kueue team publishes their guidelines. Have created a backlog task for this.